### ISSUE TITLE
Doesn't catch `content-visibility: auto;`

### ISSUE THREAD

**Issue by Volker-E** [RJWadley/stylelint-no-unsupported-browser-features#293]
Hi,
we're happily using the plugin at Wikimedia Foundation. Thanks for your work!

Weirdly a few relatively unsupported CSS properties aren't caught.
E.g. `content-visibility: auto;` with 

```
[
	"Last 3 years and Chrome > 0",
	"Last 3 years and Opera > 0",
	"Last 3 years and Edge > 0",
	"Firefox >= 91",
	"Safari >= 11",
	"iOS >= 11",
	"Android >= 5"
]
``` as browserslist support list

Or also `text-wrap: balance` ([caniuse](https://caniuse.com/css-text-wrap-balance)).

Any clue what's behind? Code can be found at [Codex, design system for Wikimedia](https://github.com/wikimedia/design-codex).

**Comment by ReneBrandt** [comment]
Hi, I am by no means an expert, but I ran into the same thing on a largely fresh `stylelint` setup. I had a similar issue where `text-wrap: wrap` wasn't reported as unsupported. I did some digging around:

* there have been multiple similar issues reported, and [the response always boils down to this](https://github.com/RJWadley/stylelint-no-unsupported-browser-features/issues/276#issuecomment-1563163699): `doiuse` does the actual flagging. If `doiuse` can't find it, `stylelint` can't as well. This is also part of the issue template when you create a new issue
* I have checked your cases (I was already busy with mine) against `doiuse`, and:
    * `text-wrap` isn't supported at all by doiuse atm (even `text-wrap: fhskjdf` passes)
    * `content-visibility` at least on the most recent versions works for me using the `.browserslistrc` below

Are you sure your stylelint and/or your command is configured correctly?

```
last 3 years and Chrome > 0
last 3 years and Opera > 0
last 3 years and Edge > 0
Firefox >= 91
Safari >= 11
iOS >= 11
Android >= 5
```
